src/Resources/config/services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+    ContaoCommunityAlliance\Translator\Contao\ContaoTranslatorService:
+        arguments:
+            - '@event_dispatcher'
+
+    contao.translator:
+        class: ContaoCommunityAlliance\Translator\Contao\ContaoTranslatorService
+        factory: 'ContaoCommunityAlliance\Translator\Contao\ContaoTranslatorService:createService'
```

Comment:
IMO the factory should not produce "itself".
We should split this up into two classes.